src/modules/home/home.user/UserHomeView.tsx

```
@@ -1,7 +1,92 @@
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card"
+import { useCart } from "@/context/CardContext"
+import { useGetAllProducts } from "@/queries/products"
+import { Link } from "react-router-dom"
+
 type Props = {}
 
 const UserHomeContainer = ({}: Props) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)